emails/index.js

```
@@ -0,0 +1,56 @@
+      subject: 'Reset your account password',
+      template: 'en/reset-password'
+    },
+    ua: {
+      subject: 'Скиньте пароль для свого акаунту',
+      template: 'ua/reset-password'
+    }
+  },
+  [emailSubject.SUCCESSFUL_PASSWORD_RESET]: {
+    en: {
+      subject: 'Your password was changed',
+      template: 'en/sucessful-password-reset'
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)